README.md

```diff
@@ -35,3 +35,95 @@ console.log(binaryPacket.toString('hex'))
 
 const jsonPacket = packet.deserializeIlpPayment(binaryPacket)
 ```
+
+### Serialize/deserialize ILQP Quote Requests/Responses
+
+#### QuoteLiquidityRequest
+
+```js
+const packet = require('ilp-packet')
+
+const binary = packet.serializeIlqpLiquidityRequest({
+  requestId: 'c217449a-69e9-4a08-9575-6e7eafa818c0',
+  destinationAccount: 'example.nexus.bob',
```

Comment:
> Why doesn't Alice ask for a quote to example.blah.dest. if she knows all of those accounts share a prefix? Or is the idea that she doesn't really know if all of them will have the same rates until the connector tells her that?

Exactly.

> Am I right in thinking that if I ask for a quote to a specific account I could ignore the prefix altogether and just use the curve?

Yes.

>  It sounds like the prefix is just extra info that I may want to use to avoid making subsequent quote requests. If that's the case, what about a name like appliesToPrefix?

Sounds good to me!

@sentientwaffle Heads up, going to rename this to `appliesToPrefix`.